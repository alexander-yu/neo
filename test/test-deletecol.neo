void main(){
    create matrix<int> a = [[1,2],[3,4],[5,6]];
    println(idelete_col(a, 1));
}