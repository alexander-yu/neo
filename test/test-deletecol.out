[[2, 5],
 [4, 4],
 [6, 2]]
[[1.5, 1],
 [3.1, 2],
 [5.5, 3]]
